src/config.js

```diff
@@ -90,6 +90,7 @@ module.exports = Object.assign({
       /* SEO: https://developers.google.com/structured-data/slsb-overview#markup_examples */
       /* SEO: https://developers.google.com/structured-data/site-name#markup_requirements */
       script: [
+        { src: "https://cdn.polyfill.io/v2/polyfill.min.js?features=Intl.~locale.en"},
```

Comment:
react-intl depends on built-in `intl` api, which is missing in old browser and safari. This polyfill add missing `intl` for those browsers.  

https://github.com/yahoo/react-intl/wiki#i18n-in-javascript